docs/content.zh/docs/connectors/datastream/dynamodb.md

```diff
@@ -0,0 +1,170 @@
+---
+title: DynamoDB
+weight: 5
+type: docs
```

Comment:
do we need aliases? 

https://github.com/apache/flink-connector-elasticsearch/blob/main/docs/content.zh/docs/connectors/datastream/elasticsearch.md?plain=1#L5

Reply:
Not sure, copied firehose and did not have one. I will add.